### PR TITLE
for camelCasing Issue

### PR DESCRIPTION
**Pull request by Aditay2002**
1)Modify the configRunner script environment to include an additional environment variable that enables camelCase aliasing.
2)Update the configRunner configuration in scriptrunner.NewScriptRunner to pass the scriptEnv.

### CODE REVIEW

File: cli/commands/generate.go [hunk 1]
```diff
@@ -52,17 +52,20 @@ var generateCmd = &cobra.Command{
...
-
-		configRunner := scriptrunner.NewScriptRunner(&scriptrunner.Config{
-			Name:          "config-runner",
-			Executable:    "node",
-			ScriptArgs:    []string{configOutFile},
-			AbsWorkingDir: wunderGraphDir,
-			Logger:        log,
-			ScriptEnv:     scriptEnv,
-			Streaming:     true,
-		})
```
Comment (Aditay2002) [inline cli/commands/generate.go:65]: removed this and modified later

File: cli/commands/generate.go [hunk 2]
```diff
@@ -52,17 +52,20 @@ var generateCmd = &cobra.Command{
...
+	configRunner := scriptrunner.NewScriptRunner(&scriptrunner.Config{
+    Name:          "config-runner",
+    Executable:    "node",
+    ScriptArgs:    []string{configOutFile},
+    AbsWorkingDir: wunderGraphDir,
+    Logger:        log,
+    ScriptEnv:     scriptEnv, // Pass the scriptEnv variable
+    Streaming:     true,
+})
+
```
Comment (Aditay2002) [inline cli/commands/generate.go:68]: added a scriptENV

File: cli/commands/generate.go [hunk 3]
```diff
@@ -205,23 +208,27 @@ var generateCmd = &cobra.Command{
...
-			AbsWorkingDir: wunderGraphDir,
-			EntryPoints:   []string{configEntryPointFilename},
-			OutFile:       configOutFile,
-			Logger:        log,
-			IgnorePaths: []string{
-				"generated",
-				"node_modules",
-			},
-			OnAfterBundle: onAfterBuild,
-		})
```
Comment (Aditay2002) [inline cli/commands/generate.go:220]: removed and modified later

File: cli/commands/generate.go [hunk 4]
```diff
@@ -205,23 +208,27 @@ var generateCmd = &cobra.Command{
...
+		"generated",
+		"node_modules",
+	},
+	OnAfterBundle: onAfterBuild,
+})
+
+// Enable camelCase aliasing for generated GraphQL fields
+configBundler.EnableCamelCaseAlias(true)
+
+err = configBundler.Bundle()
```
Comment (Aditay2002) [inline cli/commands/generate.go:228]: here code is enabling camelcase aliasing for generated graphQL fields

**Review by Aditay2002 — COMMENTED** [review]
self-review done.

**Comment by fiam** [comment]
@Aditay2002 Thanks for your contribution! However, on first look it seems it doesn't seem to compile and there's also an unneeded indentation change. Could you clarify what's the status of the PR?

**Comment by fiam** [comment]
Hello @Aditay2002 This has been opened for a while, but it seems there's nothing in here done except for formatting changes. We asked for clarification a month ago but we haven't heard from you, so we're closing this PR in the meantime. Please, let us know if plan to move ahead tackling this issue and we can reopen it. Thanks!

**Pull request closed without merging**